derive/src/lib.rs

```diff
@@ -214,13 +226,18 @@ fn generate_c_like_enum_def(variants: &VariantList, scale_info: &Ident) -> Token
         .map(|(i, v)| {
             let name = &v.ident;
             let discriminant = utils::variant_index(v, i);
+            let docs = utils::get_doc_literals(&v.attrs);
             quote! {
-                .variant(stringify!(#name), #discriminant as u64)
+                .variant(
+                    :: #scale_info ::Variant::builder(::core::stringify!(#name))
+                        .index(#discriminant as u64)
```

Comment:
```suggestion
                        .index(#discriminant as ::core::primitive::u64)
```

Reply:
this is an instance where you could apply the full path usage from Rust's `core::primitive` module.

Reply:
Ah ha. Thanks!